gr-zeromq/lib/pull_msg_source_impl.h

```diff
@@ -24,7 +24,7 @@
 #define INCLUDED_ZEROMQ_PULL_MSG_SOURCE_IMPL_H
 
 #include <gnuradio/zeromq/pull_msg_source.h>
-#include <zmq.hpp>
+#include "zmq_common_impl.h"
```

Comment:
Is including an external `_impl` header the right thing?

Reply:
This is a local `_impl` header file, created by the PR ...

Reply:
It, in turn, does `#include <zmq.hpp>` ...